plugins/janus_sip.c

```diff
@@ -4521,16 +4522,35 @@ static void *janus_sip_handler(void *data) {
 				content_type = json_string_value(content_type_text);
 
 			const char *msg_content = json_string_value(json_object_get(root, "content"));
+			char custom_headers[2048];
+			janus_sip_parse_custom_headers(root, (char *)&custom_headers, sizeof(custom_headers));
```

Comment:
Since you added custom headers support, the `sipmessage_parameters` validation should include that too, otherwise a malformed `headers` property could cause issues.

Reply:
Done.